Twig/I18nRoutingExtension.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace JMS\I18nRoutingBundle\Twig;
+
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * @author Tobias Nyholm
+ */
+class I18nRoutingExtension extends \Twig_Extension
+{
+    /**
+     * @var RequestStack requestStack
+     */
+    private $requestStack;
+
+    /**
+     * @var array locales
+     */
+    private $locales;
+
+    /**
+     * @param RequestStack $requestStack
+     * @param array        $locales
+     */
+    public function __construct(RequestStack $requestStack, $locales)
```

Comment:
Sorry, you are correct. Should I get the request from the TwigEnvironment instead?